ecl/resource_ecl_mlb_policy_v1.go

```diff
@@ -311,11 +405,30 @@ func resourceMLBPolicyV1UpdateConfigurations(d *schema.ResourceData, client *ecl
 			createStagedOpts.DefaultTargetGroupID = d.Get("default_target_group_id").(string)
 		}
 
+		if d.HasChange("backup_target_group_id") {
+			isConfigurationsUpdated = true
+			createStagedOpts.BackupTargetGroupID = d.Get("backup_target_group_id").(string)
+		}
+
 		if d.HasChange("tls_policy_id") {
 			isConfigurationsUpdated = true
 			createStagedOpts.TLSPolicyID = d.Get("tls_policy_id").(string)
 		}
 
+		if d.HasChange("server_name_indications") {
+			isConfigurationsUpdated = true
+			serverNameIndications := make([]policies.CreateStagedOptsServerNameIndication, len(d.Get("server_name_indications").([]interface{})))
```

Comment:
[NITS]
細かくて申し訳ないのですが、空行を入れて頂けると助かります

```suggestion
			isConfigurationsUpdated = true

			serverNameIndications := make([]policies.CreateStagedOptsServerNameIndication, len(d.Get("server_name_indications").([]interface{})))
```

Reply:
空行を入れるよう修正しました